scipy/sparse/sparsetools/sparsetools.cxx

```diff
@@ -376,6 +378,9 @@ call_thunk(char ret_spec, const char *spec, thunk_t *thunk, PyObject *args)
     case 'i':
         return_value = PyInt_FromSsize_t(ret);
         break;
+    case 'l':
+        return_value = PyInt_FromLong(ret);
```

Comment:
This is not the right one I think, will fail on windows.
FromLongLong is safe, since that's always at least 64-bit.

Reply:
TBH, also the call above it should be FromLongLong because `'i'` can be 64-bit. Can you fix that at the same time?